uber/decorators.py

```diff
@@ -279,8 +274,72 @@ def __call__(self, klass):
 
 
 register = template.Library()
+
+
 def tag(klass):
     @register.tag(klass.__name__)
     def tagged(parser, token):
         return klass(*token.split_contents()[1:])
     return klass
+
+
```

Comment:
Where would such functions belong? Were you thinking of putting them in different sections, like preregistration.py and groups.py?